diff --git a/src/backend/pwroutefiltermodel.rs b/src/backend/pwroutefiltermodel.rs
index ce565f0..3733b54 100644
--- a/src/backend/pwroutefiltermodel.rs
+++ b/src/backend/pwroutefiltermodel.rs
@@ -68,7 +68,9 @@ mod imp {
                         .skip(position as usize)
                         .map_while(Result::ok)
                         .enumerate() {
-                        if routeobject.direction() == widget.direction() && routeobject.availability() == ParamAvailability::Yes {
+                        if (routeobject.direction() == widget.direction() && 
+                            (routeobject.availability() == ParamAvailability::Yes || 
+                            routeobject.availability() == ParamAvailability::Unknown)) {
                             hashset.insert(a as u32);
                         }
                     }
